### DISCUSSION TITLE
Welcome to PostSRSd Discussions!

### DISCUSSION THREAD

**Discussion by roehling** [roehling/postsrsd#113] [General]
## 👋 Welcome!

This is the place where you can share your Postfix/PostSRSd configuration tips and tricks with our community, or ask questions on how to deploy it. If you have found a bug in PostSRSd, this is also the place where you can double-check with others whether this is a genuine defect or problem with your particular setup.

**Comment by slowphil** [comment]
I have a question regarding rewriting only forwarded emails. As far as I can see, there are two options for achieving this:

- On the front README it is suggested to use [Postforward](https://github.com/zoni/postforward) (which requires building the program first).

- In [Q&A#76](https://github.com/roehling/postsrsd/discussions/76) it seems this can be handled by merely a proper configuration of postfix with postsrsd.

Are the two approaches really functionally equivalent? 
If yes, shouldn't the second approach be mentionned on the README too?
If not, what are the differences and what would favour choosing one or the other?

**Reply by roehling** [reply]
As far as I recall, there have been some unresolved issues with people being unable to replicate a working configuration. You are very welcome to try the #76 approach if you are in an experimental mood. I would certainly welcome more feedback.
